blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolder.java

```diff
@@ -0,0 +1,35 @@
+package io.jenkins.blueocean.rest.impl.pipeline;
+
+import hudson.model.Item;
+import io.jenkins.blueocean.rest.Reachable;
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.service.embedded.rest.PipelineFolderImpl;
+import org.kohsuke.stapler.export.Exported;
+
+public abstract class OrganizationFolder extends PipelineFolderImpl {
```

Comment:
There is bit of duplicate compared to PR 573, https://github.com/jenkinsci/blueocean-plugin/pull/573. See https://github.com/jenkinsci/blueocean-plugin/blob/story/JENKINS-38847/blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderPipelineImpl.java. I think we can fold it in PR 573?